pkg/pillar/scripts/device-steps.sh

```diff
@@ -29,6 +29,21 @@ export LOG_TO_SYSLOG
 echo "$(date -Ins -u) Starting device-steps.sh"
 echo "$(date -Ins -u) EVE version: $(cat $BINDIR/versioninfo)"
 
+# wait for rsyslogd to be available for loggin
+LOOP_COUNT=0
+while true; do
+    logger -s ""
+    if [ $? = 0 ]; then
+        break
+    fi
+    LOOP_COUNT=$((LOOP_COUNT + 1))
+    if [ "$LOOP_COUNT" -ge 10 ]; then
+        echo "$(date -Ins -u) Error: rsyslogd did not start"
+		break
+    fi
+	echo "$(data -Ins -u) Waiting for rsyslogd"
+    sleep 1
+done
 /usr/bin/logread -F -socket /hostfs/var/run/memlogdq.sock | logger &
 
```

Comment:
This, on the other hand, needs to move to rsyslog container to make sure that we have full control over when and how things are scooped up to rsyslog.

@gkodali-zededa and I are discussing how to actually move it all the way into rsyslog.conf as a plugin -- but that's an implementation detail of rsyslog container -- the point being this line no longer belongs in pillar (btw, we should also make sure pillar drops FROM dependency on memlogd)

Reply:
@rvs I see you already moved this to rsyslog container as part of another PR.